packages/colorpickers/src/styled/ColorPicker/StyledAlphaRange.ts

```diff
@@ -29,14 +29,14 @@ const background = (props: IRGBColor & ThemeProps<DefaultTheme>) => {
   });
 };
 
-export const StyledAlphaRange = styled(StyledRange as 'input').attrs<IRGBColor>(props => ({
+export const StyledAlphaRange = styled(StyledRange as 'input').attrs<IRGBColorProps>(props => ({
   style: {
     backgroundSize: 'auto',
     background: background(props)
   },
```

Comment:
Out of curiosity, why can't this live in the style block below?

Reply:
I think this approach avoids dynamically creating a new class every time the color props change. `styled-component` will warn you when that happens.

Reply:
💀